src/core/emailParser.ts

```diff
@@ -0,0 +1,30 @@
+import { Login } from '../model/message.model'
+import { emailProviders } from './emailProviders'
+import { ImapConfig } from '../model/imapConfig.model';
+import { EmailProvider } from 'src/model/emailProvider.model';
+
+export function emailParser(login: Login): ImapConfig | undefined {
```

Comment:
I think the name is misleading, this seems to return a config, not a parser?

Reply:
Yes, I was confused about the function name, but I renamed the name to `emailConfigure`. If you prefer another name, please tell me.

Reply:
I think emailConfigure implies that it configures something whereas this function from what I can see simply creates a config that, I assume, will be used later.
But that's nitpicking so let's ignore this for now.